src/main/java/uk/ac/cam/cl/dtg/segue/api/QuestionFacade.java

```diff
@@ -232,6 +261,69 @@ public Response getQuestionsAnswered(@Context final HttpServletRequest request,
     }
   }
 
+  /**
+   * REST end point to provide five questions in random located in the question tile of the student dashboard.
+   *
+   * @param request  - this allows us to check to see if a user is currently logged in.
+   * @param subjects - a comma separated list of subjects
+   * @return a Response containing a gameboard object or containing a SegueErrorResponse.
+   */
+  @GET
+  @Path("/randomQuestions")
```

Comment:
It looks like the API tends to use underscores, rather than camel case, in the URLs.

Reply:
But to be honest, given that this is already under `questions/`, you could call this simply `random`.